Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1463,7 +1463,18 @@ void CClientVehicle::SetWheelStatus(unsigned char ucWheel, unsigned char ucStatu
                 m_pVehicle->GetDamageManager()->SetWheelStatus((eWheelPosition)(ucWheel), ucGTAStatus);
 
                 // Update the wheel's visibility
-                m_pVehicle->SetWheelVisibility((eWheelPosition)ucWheel, (ucStatus != DT_WHEEL_MISSING));
+                SString componentName;
+
+                if (ucWheel == FRONT_LEFT_WHEEL)
```

Comment:
Typically we use switch case for this.
Also this part can be described as function 'GetComponentNameForWheel'. This will improve readability, because SetWheelStatus will have less code/context.